hazelcast/src/test/java/com/hazelcast/kubernetes/KubernetesApiEndpointProviderTest.java

```diff
@@ -27,188 +27,209 @@ public KubernetesApiEndpointProviderTest() {
     public String getEndpointsResponseWithServices() {
         //language=JSON
         return "{\n"
-          + "  \"kind\": \"EndpointsList\",\n"
```

Comment:
Since you've touched these huge JSON strings, would it be ok for you to refactor it to smaller methods like the ones that we have in other tests?
Ideally we should get rid of these string one by one, as they are unmaintainable

Reply:
In my opinion, it is better to keep them as they are. These test cases are testing how parsers work, and I would prefer to see the input content presented as a raw string, similar to a static test file.

Reply:
Well, the parser will still be tested, as those methods still produce the JSON. It's just more readable and maintainable. Using such string makes it impossible to refactor it and really hard to maintain